### ISSUE TITLE
PaginatedPane's offset in Gui is doubled

### ISSUE THREAD

**Issue by Plajer** [stefvanschie/IF#16]
Encountered the issue some time ago but forgot to mention it.

PaginatedPane offset doesn't work properly, when moved either on x or y axis it's moved by double value.

**Example:**
```
        Gui gui = new Gui(plugin, 4, "test");
        PaginatedPane pane = new PaginatedPane(1, 1, 2, 2);
        gui.addPane(pane);
        pane.populateWithItemStacks(new ArrayList<ItemStack>() {{
          for(int i = 0; i < 10; i++) {
            add(new ItemStack(Material.DIRT));
          }
        }});
```
**Expected:** https://i.imgur.com/daLMk48.png
**Got:** https://i.imgur.com/KfzWBQ0.png x and y axis are in 2 slots offset not one

When setting x/y to 2 it goes 4 also.

Tested on StaticPane and OutlinePane, works good, broken only on this one.

**Comment by stefvanschie** [comment]
Thanks for your report. I was just working on that method anyway, so it will be fixed in version 0.5.4.

**Comment by stefvanschie** [comment]
This issue has been fixed in version 0.5.4; feel free to re-open this issue if this is not the case.
